src/shared-components/dropdown/index.tsx

```diff
@@ -42,7 +42,7 @@ type DropdownProps = {
 export const Dropdown = ({
   borderRadius = '4px',
   onChange,
-  options,
+  options = [{ value: undefined, label: '' }],
```

Comment:
Even though options is required, this was the default value inside Desktop and Mobile Components so to DRY I placed it up here and we will always pass at least this to those components